ibis/tests/test_impala_e2e.py

```diff
@@ -125,6 +125,45 @@ def test_create_exists_drop_view(self):
         self.con.drop_view(tmp_name)
         assert not self.con.exists_table(tmp_name)
 
+    @pytest.mark.xfail
+    def test_create_exists_drop_function(self):
+        location = ENV.test_data_dir + '/udf/libudfsample.so'
+        symbol = 'HasVowels'
+        name = 'has_vowels'
+        inputs = ['string']
+        output = 'boolean'
+        db = ENV.test_data_db
+        udf_info = udf.UDFInfo(location, inputs, output, symbol, name)
+        self.con.create_udf(udf_info, None, db)
+        assert self.con.exists_udf(name, db)
+        self.con.drop_udf(name, inputs, db=db)
+        assert not self.con.exists_udf(name, db)
+
+    def test_udf_full_workflow_using_infoclass(self):
+        location = ENV.test_data_dir + '/udf/libudfsample.so'
+        symbol = 'HasVowels'
+        name = 'has_vowels'
+        inputs = ['string']
+        output = 'boolean'
+        db = ENV.test_data_db
+        udf_info = udf.UDFInfo(location, inputs, output, symbol, name)
+        try:
+            self.con.create_udf(udf_info, None, db)
+            op = udf_info.to_operation()
+            udf.add_impala_operation(op, name, db)
+
+            def _identity_test(value):
+                return op(value).to_expr()
+
+            val = ibis.literal(True)
+            expr = _identity_test(val)
+            result = self.con.execute(expr)
+            assert result == val
+        except Exception:
+            pass
```

Comment:
I believe this may suppress test failure. Can you remove the try/except and instead add the udf name to a new `self.temp_udfs` variable (see `temp_tables` elsewhere in the file) and drop the udfs in `tearDown`?